### PR TITLE
Fixes Issue #535

### PR DESCRIPTION
**Pull request by robertojrojas**
Fixes Issue #535

- Changes isEtcd condition so it only executes for CIS tests and not OpenShift tests.

- Adds `openssl`

- Adds programs for OpenShift OKD when installed using Minishift.

### CODE REVIEW

File: cfg/config.yaml [hunk 1]
```diff
@@ -25,7 +25,6 @@ master:
       - "hyperkube apiserver"
       - "hyperkube kube-apiserver"
       - "apiserver"
-      - "openshift start master api"
```
Comment (simar7) [inline cfg/config.yaml]: For my own learning: Why are these removed?

File: cmd/root.go [hunk 2]
```diff
@@ -81,7 +81,7 @@ var RootCmd = &cobra.Command{
 
 		// Etcd is only valid for CIS 1.5 and later,
 		// this a gatekeeper for previous versions.
-		if isEtcd() && validTargets(benchmarkVersion, []string{string(check.ETCD)}) {
+		if validTargets(benchmarkVersion, []string{string(check.ETCD)}) && isEtcd() {
```
Comment (simar7) [inline cmd/root.go]: Does the order matter?

File: cmd/common_test.go [hunk 3]
```diff
@@ -410,11 +491,6 @@ func fakeExecutableInPath(execFile, execCode string) (restoreFn, error) {
 		return nil, err
 	}
 
-	err = os.Chdir(tmp)
```
Comment (simar7) [inline cmd/common_test.go]: Curious why this was removed?

File: cfg/config.yaml (hunk 1 above)
Comment (robertojrojas) [reply cfg/config.yaml]: Basically, these values are specific to the OpenShift, so they belong in the **cfg/rh-0.7/config.yaml** instead.

File: cmd/root.go (hunk 2 above)
Comment (robertojrojas) [reply cmd/root.go]: Yes,  CIS 1.4 and earlier do not include `etcd.yaml`, so checking to see if it's provided by the CIS benchmark before trying to check if the host is running **Etcd** avoids misleading errors.

File: cmd/common_test.go (hunk 3 above)
Comment (robertojrojas) [reply cmd/common_test.go]: It turns out I added the by mistake. We didn't have to navigate to the newly created `tmp` directory for the logic to work as the code refers to it using absolute paths. Removing it avoids errors with other Unit Tests that rely on being executed at the root of the source tree to find directories like `cfg`, etc...

File: cfg/rh-0.7/config.yaml [hunk 4]
```diff
@@ -10,12 +10,14 @@ master:
   scheduler:
     bins:
       - "openshift start master controllers"
+      - "hyperkube kube-scheduler"
```
Comment (mrueg) [inline cfg/rh-0.7/config.yaml]: @robertojrojas this should probably be hybershift instead of hyperkube, shouldn't it?
